### ISSUE TITLE
Investigate which Tensorflow op is causing the "UserWarning: Converting sparse IndexedSlices ... to a dense Tensor of unknown shape. This may consume a large amount of memory." warning

### ISSUE THREAD

**Issue by hsm207** [RasaHQ/rasa#10226]
### Rasa Open Source version

2.8.12

### Rasa SDK version

 _No response_ 

### Rasa X version

 _No response_ 

### Python version

3.8

### What operating system are you using?

Linux

### What happened?

Running `rasa train` produces the following warning:

`tensorflow/python/framework/indexed_slices.py:447: UserWarning: Converting sparse IndexedSlices(IndexedSlices(indices=Tensor("gradients/cond_grad/gradients/cond/GatherV2_grad/Reshape_1:0", shape=(None,), dtype=int32), values=Tensor("gradients/cond_grad/gradients/cond/GatherV2_grad/Reshape:0", shape=(None,), dtype=float32), dense_shape=Tensor("gradients/cond_grad/gradients/cond/GatherV2_grad/Cast:0", shape=(1,), dtype=int32))) to a dense Tensor of unknown shape. This may consume a large amount of memory.`

This might be caused by tensorflow/tensorflow#38168

### Command / Request

 _No response_ 

### Relevant log output

 _No response_

**Comment by lizhzh8** [comment]
I have the same problem。rasa2.8.10  python3.6

site-packages/tensorflow/python/framework/indexed_slices.py:449: UserWarning: Converting sparse IndexedSlices(IndexedSlices(indices=Tensor("gradients/cond_grad/gradients/cond/GatherV2_grad/Reshape_1:0", shape=(None,), dtype=int32), values=Tensor("gradients/cond_grad/gradients/cond/GatherV2_grad/Reshape:0", shape=(None,), dtype=float32), dense_shape=Tensor("gradients/cond_grad/gradients/cond/GatherV2_grad/Cast:0", shape=(1,), dtype=int32))) to a dense Tensor of unknown shape. This may consume a large amount of memory.
  "shape. This may consume a large amount of memory." % value)
